mantidproject/mantid
# Add gridlines and increase font size on DPW

**To test:**
Open the sans gui V2 confirm that the text is larger and that gridlines are present
Open the reflectometry gui confirm that the text is larger that gridlines are present and that you can expand into the sub trees.
Also test that the correct rows are highlighted green on process.


Fixes #21312 . 

**Release Notes** 

TODO


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
